src/arcsf/models/config_class.py

```diff
@@ -0,0 +1,58 @@
+from arcsf.config.config_class import Config
+
+
+class ModelConfig(Config):
+
+    def __init__(
+        self,
+        model_id: str,
+        model_kwargs: dict,
+        trainer_kwargs: dict,
+        use_wandb: bool,
```

Comment:
Is it necessary to have WandB stuff in the model config (could it just be in the experiment config)?

Reply:
This is now done